backpack/core/derivatives/basederivatives.py

```diff
@@ -302,6 +302,50 @@ class BaseParameterDerivatives(BaseDerivatives, ABC):
     For most layers, these shapes correspond to shapes of the module input or output.
     """
 
+    @shape_check.param_mjp_accept_vectors
+    @shape_check.param_mjp_check_shapes
+    def param_mjp(
+        self,
+        param_str: str,
+        module: Module,
+        g_inp: Tuple[Tensor],
+        g_out: Tuple[Tensor],
+        mat: Tensor,
+        sum_batch: bool = True,
+        subsampling: List[int] = None,
+    ) -> Tensor:
+        """Compute matrix-Jacobian products (MJPs) of the module w.r.t. a parameter.
+
+        Handles both vector and matrix inputs. Preserves input format in output.
+
+        Internally calls out to ``_{param_str}_jac_t_mat_prod`` function that must be
+        implemented by descendants. It follows the same signature, but does not have
+        the ``param_str`` argument.
+
+        Args:
+            param_str: Attribute name under which the parameter is stored in the module.
+            module: Module whose Jacobian will be applied. Must provide access to IO.
+            g_inp: Gradients w.r.t. module input.
+            g_out: Gradients w.r.t. module output.
+            mat: Matrix the Jacobian will be applied to. Has shape
+                ``[V, *module.output.shape]`` (matrix case) or same shape as
+                ``module.output`` (vector case).
+            sum_batch: Sum out the MJP's batch axis. Default: ``True``.
+            subsampling: Indices of samples along the output's batch dimension that
+                should be considered. Defaults to ``None`` (use all samples).
+
+        Returns:
+            Matrix-Jacobian products. Has shape ``[V, *param_shape]`` when batch
+            summation is enabled (same shape as parameter in the vector case). Without
+            batch summation, the result has shape ``[V, N, *param_shape]`` (vector case
+            has shape ``[N, *param_shape]``).
```

Comment:
Add the following sentence: If used with `subsampling`, the batch size `N` is replaced by `len(subsampling)`.

Reply:
Incorporated.